src/main/kotlin/net/shadowfacts/forgelin/Forgelin.kt

```diff
@@ -17,9 +17,9 @@ object Forgelin {
 	const val VERSION = "@VERSION@"
 
 	@EventHandler
-	fun onPreInit(event: FMLPreInitializationEvent) {
+	fun onPreInit(event: FMLConstructionEvent) {
```

Comment:
now now... don't be lazy!... rename 'onPreInit' to 'onConstruct' or something... :D

Reply:
Oops... Thanks for noticing. :P